lib/mindwendel_web/live/idea_live/index_component.html.heex

```diff
@@ -3,108 +3,16 @@
     id="ideas"
     phx-hook="Sortable"
     class="mb-5 sortable d-flex flex-wrap justify-content-start"
-    data-sortable-enabled={to_string(has_move_permission(@brainstorming, @current_user))}
   >
     <%= for idea <- @ideas do %>
       <div
         class="card-mindwendel m-3 shadow-sm p-2 rounded IndexComponent__IdeaCard"
-        role={if has_move_permission(@brainstorming, @current_user), do: "button", else: ""}
         data-testid={idea.id}
         data-id={idea.id}
         data-brainstorming-id={idea.brainstorming_id}
         data-position={idea.position_order}
       >
         <div class="card-body-mindwendel-idea">
-          <%= if @current_user.id in [idea.user_id | @brainstorming.moderating_users |> Enum.map(& &1.id)] do %>
```

Comment:
a bit confused how we can delete all that? :thinking:

Reply:
we definitely can't but it dawns on me that maybe it was done because of the lanes but I'm not sure